conn-sdk-cli/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2024 Meroxa, Inc.
```

Comment:
Maybe 2025 here?

Reply:
New laptop, I need to set up that the header is added automatically 🙈